views/index.html

```diff
@@ -38,13 +38,16 @@
           route: "home",
           state: {},
           heartbeat: { timestamp, interval: ONE_SECOND * 3 },
-          widgets: { order: [], timestamp, interval: ONE_MINUTE * 5, items: [] },
```

Comment:
Items were removed since they were never used and we store them under the `state` property